infrastructure/build_and_push.sh

```diff
@@ -14,9 +14,9 @@ ACCOUNT_ID="129033205317"
 CLUSTER_NAME="c12-ecs-cluster"
 SERVICE_NAME="c12-energy-dashboard"
 REPO_PREFIX="c12-energy"
-REPO_SUFFIX=("extract" "extract-carbon" "pipeline" "dashboard")
-IMAGE_NAMES=("extract_to_s3" "extract_carbon" "transform_load" "dashboard")
-DOCKERFILES=("Dockerfile.extract_many" "Dockerfile.extract_carbon" "Dockerfile.transform_load" "Dockerfile.dashboard")
+REPO_SUFFIX=("extract" "extract-carbon" "pipeline" "dashboard", "email-service")
```

Comment:
This is why I went with a loop, you just add the details to the lists and it builds it for you. Pleased to see this script come in handy.